tree/30_generic_methods/file_ensure_lines_present.cf

```diff
@@ -30,25 +30,30 @@
 bundle agent file_ensure_lines_present(file, lines)
 {
   vars:
-      "old_class_prefix" string => canonify("file_ensure_lines_present_${file}");
-      "promisers"          slist => { @{this.callers_promisers}, cf_null }, policy => "ifdefined";
-      "class_prefix"      string => canonify(join("_", "promisers"));
+      "old_class_prefix"  string => canonify("file_ensure_lines_present_${file}");
+
       "args"               slist => { "${file}", "${lines}" };
+      "report_param"      string => join("_", args);
+      "full_class_prefix" string => canonify("file_lines_present_${report_param}");
```

Comment:
shouldn't it be `file_ensure_lines_present`?

Reply:
you are totally right